### DISCUSSION TITLE
抓取 隐私等级 信息

### DISCUSSION THREAD

**Discussion by sabersalv** [ronggang/PT-Plugin-Plus#769] [Ideas]
为了给隐私需要的人，不需要每个网站点进去查看

**Comment by ted423** [comment]
我觉得这个功能，怎么说，有点奇怪。
大部分都要衡量投入和产生的价值。
隐私等级这种，大部分不会天天看的。真有需要就一个一个点呗

**Reply by sabersalv** [reply]
确实是一个小众的需求, 主要用于这点
求药的时候一般需要把隐私设置调低, 求到之后再把隐私调回去, 这个时间不确定, 1个星期到1个月都有可能. 时间久了，站点多了，很容易就忘记哪个站点没有调回去。
